### DISCUSSION TITLE
metrics not showing up in UI

### DISCUSSION THREAD

**Discussion by vkstanwar** [lightdash/lightdash#5902] [Q&A]
created metrics in demo jaffle_shop  , executed the deployed the code still the metrics dont show up in UI

**Comment by TuringLovesDeathMetal** [comment]
@vkstanwar would you mind expanding a bit more around what happened here? 

Were the metrics using the Lightdash format, or the dbt format? 

Are you on dbt version 1.5? (Lightdash currently only supports up to dbt 1.4 because there were breaking changes introduced in 1.5, but we're working to implement it this week 🤞 )
